libdevcore/Log.h

```diff
@@ -73,10 +73,15 @@ BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_debugLogger,
     (boost::log::keywords::severity = 0)(boost::log::keywords::channel = "debug"))
 #define cdebug LOG(dev::g_debugLogger::get())
 
-BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_noteLogger,
+BOOST_LOG_INLINE_GLOBAL_LOGGER_CTOR_ARGS(g_note2Logger,
     boost::log::sources::severity_channel_logger_mt<>,
-    (boost::log::keywords::severity = 1)(boost::log::keywords::channel = "note"))
-#define cnote LOG(dev::g_noteLogger::get())
+    (boost::log::keywords::severity = 2)(boost::log::keywords::channel = "note2"))
```

Comment:
You are changing with this a lot of messages going into `cnote`. I'm not sure it's a good idea. What are you trying to achieve?

Reply:
I need another log level for test output. 
0) no output
1) boost output
2) test output (uses cnote)
3) cnote ouput
4) ctrace output

Reply:
Why are you ignoring in this all other channels, what should happen to them?

What's your idea about distinction between cnote and ctrace? What makes them different, what's the purpose of each?

> test output (uses cnote)

I would propose for these messages to go into separate test-specific channel/logger.

Reply:
(If you need just specific channels it would be much easier and better instead of dancing around verbosity to setup a special filter based on channels)

Reply:
boost output to std output. what I need is 
0)  no output at all 
1) stderr + boost output
2) stderr + boost output + more output from test helper
3) all + verbosity levels

so before I was not using cnote and ctrace. so you say you going to delete those. then I could just use the verbosity and std::cout  + if

Reply:
`std::cout` + `if` is a bad idea if I understand it correctly. boost.log is awesome at filtering your logs for you, so let's use it for this and not have an `if`s  with each log message.

So if I understand your needs correctly, you could
instead of `cnote` and `ctrace` use 2 new test-specific channels. It could be one channel with a name `test` and macros to log messages with 2 different verbosities.

for 0 & 1 - disable all boost.log logs
for 2 - enable only `test` channel
for 3 - enable channels based on verbosity (call `setupLogging` like you do now)